rdr_service/gcloud_functions/genomic_ingest_manifest_function/main.py

```diff
@@ -220,7 +220,7 @@ def run(self):
                 "cloud_function": True,
             }
 
-            bypass_raw_keys = ['aw5']
+            bypass_raw_keys = []
```

Comment:
Good catch. Just FYI, Cloud Functions are not deployed during the App Engine deploy, so we need to do this manually.

Reply:
We should just remove this list for bypassing and the accompanying conditional for the check.